cvmfs/authz/authz_fetch.cc

```diff
@@ -118,8 +122,16 @@ void AuthzExternalFetcher::ExecHelper() {
   MakePipe(pipe_recv);
   char *argv0 = strdupa(progname_.c_str());
   char *argv[] = {argv0, NULL};
-  char *envp0 = strdupa("CVMFS_AUTHZ_HELPER=yes");
-  char *envp[] = {envp0, NULL};
+
+  const bool strip_prefix = true;
+  vector<string> authz_env =
+    options_manager_->GetEnvironmentSubset("CVMFS_AUTHZ_", strip_prefix);
+  vector<char *> envp;
+  for (unsigned i = 0; i < authz_env.size(); ++i)
+    envp.push_back(strdupa(authz_env[i].c_str()));
```

Comment:
Is `strdupa` necessary here?  This should create a string object, which will allocate new memory.  No?